tests/test_postgres.py

```diff
@@ -70,7 +70,7 @@ def _generate_test_matrix() -> List[ParameterSet]:
                         ),
                         # https://github.com/SUSE/BCI-tests/issues/647
                         healthcheck_timeout=(
-                            timedelta(minutes=3)
+                            timedelta(minutes=4)
```

Comment:
If 3 was not enough, then yank it up to 6. If we are on the brick this will only lead to sporadic failures.

Reply:
Increased to 6 minutes.